src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/MethodeBodyTransformationXMLEventHandlerRegistry.java

```diff
@@ -0,0 +1,126 @@
+                        videoMatcher
+                ),
+                "iframe"
+        );
+        registerStartAndEndElementEventHandler(new PodcastXMLEventHandler(new StripElementAndContentsXMLEventHandler()), "script");
+        registerStartAndEndElementEventHandler(new RetainWithSpecificAttributesXMLEventHandler("src", "alt", "width", "height"), "img");
+
+        //timelines
+        registerStartAndEndElementEventHandler(new RetainXMLEventHandler(),
+                "timeline", "timeline-header", "timeline-credits",
+                "timeline-sources", "timeline-byline", "timeline-item", "timeline-date", "timeline-title",
+                "timeline-body"
+        );
+
+
+        // strip html5 tags whose bodies we don't want
+        registerStartElementEventHandler(new StripElementAndContentsXMLEventHandler(),
+                "applet", "audio", "base", "basefont", "button", "canvas", "caption", "col",
+                "colgroup", "command", "datalist", "del", "dir", "embed", "fieldset", "form",
+                "frame", "frameset", "head", "input", "keygen", "label", "legend",
+                "link", "map", "menu", "meta", "nav", "noframes", "noscript", "object",
+                "optgroup", "option", "output", "param", "progress", "rp", "rt", "ruby",
+                "s", "select", "source", "strike", "style", "tbody",
+                "td", "textarea", "tfoot", "th", "thead", "tr", "track", "video", "wbr"
+        );
+        // strip methode tags whose bodies we don't want
+        registerStartElementEventHandler(new StripElementAndContentsXMLEventHandler(),
+                "annotation", "byline", "editor-choice", "headline", "inlineDwc", "interactive-chart",
+                "lead-body", "lead-text", "ln", "photo", "photo-caption", "photo-group",
+                "plainHtml", "promo-headline", "promo-image", "promo-intro",
+                "promo-link", "promo-title", "promobox-body", "pull-quote", "pull-quote-header",
+                "pull-quote-text",
+                "readthrough", "short-body", "skybox-body", "stories",
+                "story", "strap", "videoObject", "web-alt-picture", "web-background-news",
+                "web-background-news-header", "web-background-news-text",
+                "web-picture", "web-pull-quote-source", "web-pull-quote-text",
+                "web-skybox-picture", "web-subhead", "web-thumbnail", "xref", "xrefs"
+        );
+
+        registerStartAndEndElementEventHandler(new SimpleTransformTagXmlEventHandler("h3", "class", "ft-subhead"), "subhead");
+        registerStartAndEndElementEventHandler(new SimpleTransformBlockElementEventHandler(new StAXTransformingBodyProcessor(this), "ft-timeline"), "timeline");
+        registerStartAndEndElementEventHandler(new ReplaceElementXMLEventHandler("div", "class"), "layout-set", "layout", "layout-slot");
+
+        registerStartAndEndElementEventHandler(new RecommendedXMLEventHandler(new RecommendedXMLParser()), "recommended");
```

Comment:
We can change registerStartAndEndElementEventHandler with registerStartElementEventHandler.